.kokoro/functions/tokenservice.cfg

```diff
@@ -3,11 +3,17 @@
 # Set the folder in which the tests are run
 env_vars: {
     key: "PROJECT"
-    value: "functions/dialogflow/functions/"
+    value: "functions/tokenservice/functions/"
 }
 
 # Tell the trampoline which build file to use.
 env_vars: {
     key: "TRAMPOLINE_BUILD_FILE"
     value: "github/nodejs-docs-samples/.kokoro/build.sh"
 }
+
+# Configure the docker image for kokoro-trampoline.
+env_vars: {
```

Comment:
Is this supposed to be added?

Reply:
@nnegrey Yes, as per the discussion with @ace-n he mentioned, migrate functions/tokenservice off the emulator and This package is deprecated: https://www.npmjs.com/package/@google-cloud/functions-emulator

also he mentioned We need to remove it from our samples to get our tests passing on Node 8.
https://github.com/GoogleCloudPlatform/nodejs-docs-samples/pull/1375 , he did a similar thing for `functions/datastore` and asked me to follow them.

#1255

Reply:
N.B: As @santhoshvaddi and I discussed offline, this will require changes in our internal Kokoro configs.